networkdb/networkdb.go

```diff
@@ -558,14 +555,23 @@ func (nDB *NetworkDB) JoinNetwork(nid string) error {
 		nodeNetworks = make(map[string]*network)
 		nDB.networks[nDB.config.NodeName] = nodeNetworks
 	}
-	nodeNetworks[nid] = &network{id: nid, ltime: ltime}
-	nodeNetworks[nid].tableBroadcasts = &memberlist.TransmitLimitedQueue{
-		NumNodes: func() int {
-			nDB.RLock()
-			defer nDB.RUnlock()
-			return len(nDB.networkNodes[nid])
+	n, ok := nodeNetworks[nid]
+	var entries int
+	if ok {
+		entries = n.entriesNumber
+	}
+	nodeNetworks[nid] = &network{
+		id:    nid,
+		ltime: ltime,
+		tableBroadcasts: &memberlist.TransmitLimitedQueue{
+			NumNodes: func() int {
+				nDB.RLock()
+				defer nDB.RUnlock()
+				return len(nDB.networkNodes[nid])
+			},
+			RetransmitMult: 4,
```

Comment:
Nit: Pls reorganize this code so that the changes doesnt popup. The only real change in this blob is the addition of `entriesNumber` and the rest of if is purely code reorganization. It is very distracting to see such changes in a PR with other sensitive changes.